packages/driver/src/cy/commands/actions/scroll.js

```diff
@@ -333,6 +338,10 @@ module.exports = (Commands, Cypress, cy, state) => {
       }
 
       const ensureScrollability = () => {
+        if (!options.ensureScrollability) {
```

Comment:
Just some comments to clarify + link to original issue.

```suggestion
        // Some elements are not scrollable, user may opt out of error checking
        // https://github.com/cypress-io/cypress/issues/1924
        if (!options.ensureScrollability) {
```